rewrite-java/src/main/java/org/openrewrite/java/style/Autodetect.java

```diff
@@ -64,24 +65,32 @@ public static Autodetect detect(List<J.CompilationUnit> cus) {
                 spacesStatistics.getSpacesStyle()));
     }
 
+    private static class FormatStatistics {
```

Comment:
`GeneralFormatStatisitcs` or `TextFormatStatistics` depending on what we name the top level style class.